panic after Timeout waiting for session RPC

wallet running latest commit ```82b34ce2fc952dbf1b181df058da7bfcd33eecff```
```
20:13:25 2017-03-08 [INF] WLLT: Successfully synchronized wallet accounts to account number 2.
20:13:25 2017-03-08 [INF] WLLT: Loading active addresses and unspent outputs...
20:13:58 2017-03-08 [INF] WLLT: Registering for transaction notifications for 215470 address(es) and 3909 output(s)
20:14:01 2017-03-08 [INF] WLLT: Fetching headers
20:14:01 2017-03-08 [INF] WLLT: Fetched 6 new header(s)
20:14:01 2017-03-08 [INF] WLLT: Rescanning blocks 285764-285769...
20:15:16 2017-03-08 [ERR] CHNS: Timeout waiting for session RPC
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8c0c74]

goroutine 33 [running]:
github.com/decred/dcrwallet/wallet.(*Wallet).syncWithChain(0xc42007e480, 0xc420804000, 0xf, 0xc420176000)
        /home/dhill/.go/src/github.com/decred/dcrwallet/wallet/wallet.go:875 +0x204
github.com/decred/dcrwallet/wallet.(*Wallet).handleChainNotifications(0xc42007e480, 0xc420804000)
        /home/dhill/.go/src/github.com/decred/dcrwallet/wallet/chainntfns.go:85 +0x39
created by github.com/decred/dcrwallet/wallet.(*Wallet).SynchronizeRPC
        /home/dhill/.go/src/github.com/decred/dcrwallet/wallet/wallet.go:422 +0x174
```

--- a/wallet/wallet.go
+++ b/wallet/wallet.go
@@ -1008,11 +1008,11 @@ func (w *Wallet) syncWithChain(chainClient *chain.RPCClient) error {
 	// startup.
 	// TODO A proper pass through for dcrrpcclient for these cmds.
 	if w.initiallyUnlocked {
-		_, err = w.chainClient.RawRequest("rebroadcastwinners", nil)
+		_, err = chainClient.RawRequest("rebroadcastwinners", nil)
 		if err != nil {
 			return err
 		}
-		_, err = w.chainClient.RawRequest("rebroadcastmissed", nil)
+		_, err = chainClient.RawRequest("rebroadcastmissed", nil)
 		if err != nil {
 			return err
 		}

